README.md

```diff
@@ -74,17 +72,15 @@ If you are working on the Fn Project, want to work on it or are creating extensi
 * Learn how to [contribute](CONTRIBUTING.md)
```

Comment:
Should link to: https://github.com/fnproject/fn/blob/master/CONTRIBUTING.md. Otherwise it looks good.

Reply:
good call -- going to move this over to docs repo right now actually... stand by for next PR.